src/recorder.js

```diff
@@ -145,7 +146,11 @@ Recorder.prototype.initWorker = function(){
           break;
         case 'page':
           this.encodedSamplePosition = e['data']['samplePosition'];
-          onPage(e['data']['page']);
+          if ( this.config.streamOpusPackets && typeof e['data'] === 'object' && e['data']['type'] === 'opus' ){
+            this.streamOpusPacket( e['data']['data'] );
```

Comment:
since `onPage` is assigned to `streamOpusPacket` above, can we just call it here and conditionalize the parameter?

```
var data = this.config.streamOpusPackets && typeof e['data'] === 'object' && e['data']['type'] === 'opus' ? e['data']['data'] : e['data']['page'];
onPage(data);
```